.gitignore

```diff
@@ -21,3 +21,5 @@
 .*.sw?
 /.yardoc/
 /Guardfile
+bolt-debug.log
```

Comment:
the file is managed via modulesync_config, can you please a PR there as well? https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.gitignore.erb

Reply:
see https://github.com/voxpupuli/modulesync_config/pull/879